Library/ViewModels/PledgeCTAContainerViewViewModel.swift

```diff
@@ -27,21 +28,42 @@ public protocol PledgeCTAContainerViewViewModelType {
 public final class PledgeCTAContainerViewViewModel: PledgeCTAContainerViewViewModelType,
   PledgeCTAContainerViewViewModelInputs, PledgeCTAContainerViewViewModelOutputs {
   public init() {
-    let project = self.configureWithValueProperty.signal
+    let projectOrError = self.projectOrErrorProperty.signal
       .skipNil()
       .filter(second >>> isFalse)
       .map(first)
 
-    self.activityIndicatorIsAnimating = self.configureWithValueProperty.signal
+    let project = projectOrError
+      .map(Either.left)
       .skipNil()
-      .map(second)
 
-    self.rootStackViewAnimateIsHidden = self.activityIndicatorIsAnimating
+    let projectError = projectOrError
+      .map(Either.right)
+      .skipNil()
+
+    self.activityIndicatorIsHidden = self.projectOrErrorProperty.signal
```

Comment:
Let's chain this to `projectOrError` as well.

Reply:
`projectOrError` only emits when the project is no longer loading. For a better readability, I will create a property `isLoading`
```
let isLoading = self.projectOrErrorProperty.signal
      .skipNil()
      .map(second)
```